### ISSUE TITLE
JSON editor: add colon automatically

### ISSUE THREAD

**Issue by fulldecent** [Microsoft/vscode#69546]
# Test case

![untitledb2](https://user-images.githubusercontent.com/382183/53503676-8731f800-3a7e-11e9-9d40-09695ccf41e6.gif)

# Expected

After typing `name` <kbd>TAB</kbd>, the editor should correct this to `"name":`.

# Actual

It is corrected to `"name"` (without `:`).

**Comment by dbeigi** [comment]
Hi @aeschli can i work on this feature?

**Comment by aeschli** [comment]
@dbeigi Yes, sure. The code is in here: https://github.com/microsoft/vscode-json-languageservice/blob/c477a20432502febb07c99c08886258659e1b1d8/src/services/jsonCompletion.ts#L244
